Inititialize/clear more variables. Now this test works on Alphas too.
--- ../testbed/testcms.c	Tue Feb 13 12:08:32 2001
+++ ../testbed/testcms.c	Wed Sep  5 10:04:07 2001
@@ -582,5 +582,5 @@
 static void ClearStats(LPSTATS p)
 {
-       p -> x = p -> y = p -> x2 = p -> y2 = p -> xy
+       p -> n = p -> x = p -> y = p -> x2 = p -> y2 = p -> xy
        = p -> Peak = 0.0;
 }
